### ISSUE TITLE
Change the date-fields from string to DateTime

### ISSUE THREAD

**Issue by TopperDEL** [TopperDEL/libstorj.NET#3]
Within the Models there are some Date-Fields like "Created" on a file - they are strings currently as they are provided as strings from the Java-libstorj-library. Convert them to DateTime for better .Net-Usability.

**Comment by kaloyan-raev** [comment]
@TopperDEL I'd appreciate if you open issues like this also in the java-libstorj project.

**Comment by TopperDEL** [comment]
You're right, sorry. Maybe this is the better place for this as the Java-consumers would like to have "real Dates", too. True. Will do this - then my wrapper can consume them natively.

**Comment by TopperDEL** [comment]
Opened an issue for this in [java-libstorj](https://github.com/Storj/java-libstorj/issues/9)
